packages/core/addon/visualization/component.ts

```diff
@@ -3,18 +3,26 @@
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  */
 import Component from '@glimmer/component';
-import RequestFragment from 'navi-core/models/request';
-import YavinVisualizationModel from 'navi-core/models/visualization-v2';
-import NaviFactResponse from 'navi-data/models/navi-fact-response';
+// eslint-disable-next-line ember/no-classic-components
+import type EmberComponent from '@ember/component';
+import type RequestFragment from 'navi-core/models/request';
+import type YavinVisualizationModel from 'navi-core/models/visualization-v2';
+import type NaviFactResponse from 'navi-data/models/navi-fact-response';
+import type { YavinVisualizationManifest } from './manifest';
 
 export interface YavinVisualizationArgs<Settings = unknown> {
   request: RequestFragment;
   response: NaviFactResponse;
   settings: Settings;
   isEditing: boolean;
   isReadOnly: boolean;
+  isPrint: boolean;
   onUpdateReport: (action: string, ...params: unknown[]) => void;
   onUpdateSettings(model: YavinVisualizationModel<Settings>): void;
+  // temporary legacy support
+  manifest: YavinVisualizationManifest;
```

Comment:
Could this be a private field in the vis component?

Reply:
This seems to be the cleanest way for now other than potentially registering a bunch of new wrapper components with their manifests set